tests/test_simpleitk.py

```diff
@@ -14,10 +14,16 @@
 test_dir = get_test_dir()
 
 
+itk = None
 try:
-    import SimpleITK as itk
+    import itk
 except ImportError:
-    itk = None
+    pass
+try:
+    if not itk:
+        import SimpleITK as itk
+except ImportError:
+    pass
```

Comment:
How hard would it be to test both itk and simpleitk?
* Can we pip-install both side by side?
* Can we somehow force imageio to use one over the other?

Or is the SimpleItk functionality so thin that we can rely on testing just one?

Reply:
I tested locally with `itk`, `simpleitk`, and with both installed. If both are installed, it will prefer `itk`. There is not currently a way to force imageio to use one or the other, but I do not see a big need for it.

I tried a few times to change the TravisCI installation, but I ran into odd failures.

Reply:
Fair enough!